pkg/log/log.go

```diff
@@ -147,11 +147,11 @@ func (e LogError) Error() string {
 }
 
 func (l FilteredLogger) Msg(msg interface{}) {
-	l.log(2, "msg", msg)
+	l.log(3, "msg", msg)
 }
 
 func (l FilteredLogger) Msgf(msg string, args ...interface{}) {
-	l.log(2, "msg", fmt.Sprintf(msg, args...))
+	l.log(3, "msg", fmt.Sprintf(msg, args...))
```

Comment:
I think that if you are now using Msgf or Msg directly, then it will show the wrong frame. for Errorf, ... we need here 3, for direct invocatoins 2.

Reply:
@rmohr I checked code and not found direct call to `Msgf` or to `Msg`, except calls in the `log.go` file and `log_test.go`.
I believe I can travel on the stack and check `callers` names, but for me looks more nice way just to restrict to import `Msgf` and `Msg`.
What do you think?

Reply:
Yes please make `msgf` and `msg` private that would work for me.